be/src/storage/delete_handler.cpp

```diff
@@ -217,12 +217,12 @@ bool DeleteHandler::parse_condition(const std::string& condition_str, TCondition
 
     try {
         // Condition string format
-        // eg:  condition_str="c1 = 1597751948193618247  and length(source)<1;\n;\n"
+        // eg:  condition_str="c1= 1597751948193618247  and length(source)<1;\n;\n"
         //  group1:  ([^\0=<>!\*]{1,64}) matches "c1"
         //  group2:  ((?:=)|(?:!=)|(?:>>)|(?:<<)|(?:>=)|(?:<=)|(?:\*=)|(?:IS)) matches  "="
-        //  group3:  ((?:[\s\S]+)?) matches "1597751948193618247  and length(source)<1;\n;\n"
+        //  group3:  ((?:[\s\S]+)?) matches " 1597751948193618247  and length(source)<1;\n;\n"
```

Comment:
out of curious, why use string to save sub conditions instead of a well defined structure, e.g. pb to save this.

Reply:
historical reason, and pb is used in cloud native table.